assets/prometheus-k8s/rules.yaml

```diff
@@ -92,6 +92,11 @@ spec:
     - expr: |
         sum(rate(container_cpu_usage_seconds_total{job="kubelet", metrics_path="/metrics/cadvisor", image!="", container!="POD"}[5m])) by (namespace)
       record: namespace:container_cpu_usage_seconds_total:sum_rate
+    - expr: |
+        sum by (cluster, namespace, pod, container) (
+          rate(kube_pod_container_status_last_terminated_reason{reason="OOMKilled",job="kube-state-metrics"})
+        ) * on (cluster, namespace, pod) group_left(node) max by(cluster, namespace, pod, node) (kube_pod_info)
+      record: node_namespace_pod_container_status:oom_total
```

Comment:
Best practices dictate to have a name of the aggregator at the end of the recording rule. In this case it might be better to have something like `kube_pod_container_status_last_terminated_reason:oom_total:sum`

Reply:
thanks... fixed.